docs/client-concepts/certificates/working-with-certificates.asciidoc

```diff
@@ -116,7 +115,7 @@ the local CA certificate is part of the chain that was used to generate the serv
 ==== Client Certificates
 
 X-Pack also allows you to configure a {xpack_current}/pki-realm.html[PKI realm] to enable user authentication
-through client certificates. The {ref_current}/certutil.html[+certutil+ tool] included with X-Pack allows you to
+through client certificates. The {ref_current}/certutil.html`certutil` tool] included with X-Pack allows you to

```

Comment:
This change will break the documentation; it needs to be fixed in https://github.com/russcam/asciidocnet. Should revert this change